.github/workflows/cherry-pick-since-release-branch.yml

```
@@ -124,7 +124,7 @@ jobs:
   # This job runs after all cherry-pick attempts and reports the final status
   report_status:
     needs: [get-target-release-branches, release_pull_request]
-    if: needs.get-target-release-branches.outputs.branches != '[]'
+    if: always() && needs.get-target-release-branches.outputs.branches != '[]'
```

What about 
```suggestion
    if: always() && needs.get-target-release-branches.result == 'success' && needs.get-target-release-branches.outputs.branches != '[]'
```

I'm afraid if `get-target-release-branches` is skipped, we get `null` (or empty?) output instead of `[]`, then this will still run.

However,

```
const prNumber = ${{ needs.get-target-release-branches.outputs.pr_number }};
```
will be interpolated as
```
const prNumber = ;
```
lead to syntax error.